app/Support/BackupManager.php

```
@@ -0,0 +1,620 @@
+
+    /**
+     * Import a database.sql dump via multi_query. Runs on a DEDICATED
+     * connection: the request connection may have just produced the unbuffered
+     * pre-restore dump, after which a multi_query on the same handle can report
+     * success yet silently not persist. A fresh connection avoids that.
+     */
+    private function importDatabase(string $sqlPath): void
+    {
+        $sql = file_get_contents($sqlPath);
+        if ($sql === false || $sql === '') {
+            throw new \RuntimeException(__('Dump del database vuoto o illeggibile'));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '396', column '23'). (undefined)

(MissingImport)